db/migrate/20230901082957_rename_columns_in_posts.rb

```diff
@@ -0,0 +1,6 @@
+class RenameColumnsInPosts < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :posts, :commentCounter, :comments_counter
+    rename_column :posts, :likesCounter, :likes_counter
+  end
+end
```

Comment:
![image](https://github.com/AleWaweru/blog-App/assets/93039359/3280bf6c-9ca7-4999-97b6-65ae2e2f761f)

- [ ] Great work on making the changes so far but I guess there is an issue :fearful: Kindly fix this migrations so the next reviewer can view the changes you made according to the previous reviewer :+1:

Reply:
@HAMMAS-SALEEM  done